src-java/flowmonitoring-topology/flowmonitoring-messaging/src/main/java/org/openkilda/messaging/info/haflow/UpdateHaSubFlowCommand.java

```diff
@@ -0,0 +1,66 @@
+/* Copyright 2023 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.messaging.info.haflow;
+
+import org.openkilda.messaging.command.CommandData;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+
+/**
+ * Represents update ha flow in monitoring command.
+ */
+@Value
+@EqualsAndHashCode(callSuper = false)
+@JsonSerialize
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class UpdateHaSubFlowCommand extends CommandData {
+    private static final long serialVersionUID = 1L;
+
+    @JsonProperty("flow_id")
+    String flowId;
+
+    @JsonProperty("max_latency")
+    Long maxLatency;
+
+    @JsonProperty("max_latency_tier_2")
```

Comment:
seems like this style with _ before 2 is used only in this single place. I think it worth to be consistent with other usages if possible. 
Is there any reason for having this annotations here?

Reply:
UpdateHaSubFlowCommand class is a clone of UpdateFlowCommand and exactly to be consistent with the code style in the package it has annotations and  _ before 2.

Reply:
if it is an exception than it worth to doc it. Otherwise it looks like a mistake (and maybe it was in that single place, but now it is propagated). You can use a full text search of "max_latency_tier_2" and "max_latency_tier2" to see the difference.